opensafely/_vendor/certifi/cacert.pem

```diff
@@ -4604,3 +4282,44 @@ AZKXRRJ+oPM+rRk6ct30UJMDEr5E0k9BpIycnR+j9sKS50gU/k6bpZFXrsY3crsC
 MGclCrEMXu6pY5Jv5ZAL/mYiykf9ijH3g/56vxC+GCsej/YpHpRZ744hN8tRmKVu
 Sw==
 -----END CERTIFICATE-----
+
```

Comment:
Should this certificate be visible or can it be stashed within the repo settings as TOKEN. I've not done this with github actions but have with Travis and it seemed ok?

Reply:
Yes, we do need this. There are public root certificates which the `requests` library uses for verifying HTTPS connections. As we're vendoring in all our dependencies these get pulled in as well.

Always good to query anything that looks like a key though!

Reply:
cool thanks for the explanation